drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java

```diff
@@ -64,7 +65,7 @@ public String getSystemBinaryProperty() {
     private class ChromeStorageSources extends GoogleStorageSource {
 
         ChromeStorageSources(String baseUrl) {
-            super(baseUrl, baseUrl + "LATEST_RELEASE");
+            super(baseUrl, baseUrl + "LATEST_RELEASE", new HttpClient()); // TODO
```

Comment:
Leftover from overall question of passing new to objects, as your comments above.